ShadowsocksX-NG/example-gui-config.json

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
Because you haven't added it to project, so it not added to 'Copy Bundle Resources' build phase.